config/pipelines/Jenkinsfile-contra-lib-sample/JenkinsfilePRBuild

```diff
@@ -0,0 +1,90 @@
+/*
+This requires a keytab to run the rpm-build step.
```

Comment:
Do you need a keytab from Fedora infra or can you also use your own kerb creds?

Reply:
You can use your own. I updated the docs.